### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mortonjt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Install
 =======
 
 ```
-conda install -c biocore VSEARCH MAFFT SortMeRNA numpy
+conda install -c biocore deblur
```
Comment (wasade) [inline README.md]: testing this now. btw, i noticed that the deblur package from conda is ~85MB. which is really large. the repo isn't that size, so was a bit surprised. Any idea why its so large?

Comment (wasade) [inline README.md]: Possible my env is messed up on barnacle, but I'm getting:

```bash
$ deblur
Traceback (most recent call last):
  File "/home/mcdonadt/miniconda3/envs/deblur/bin/deblur", line 4, in <module>
    __import__('pkg_resources').run_script('deblur==0.0.1.dev0', 'deblur')
  File "/home/mcdonadt/miniconda3/envs/deblur/lib/python3.5/site-packages/setuptools-25.1.6-py3.5.egg/pkg_resources/__init__.py", line 719, in run_script
  File "/home/mcdonadt/miniconda3/envs/deblur/lib/python3.5/site-packages/setuptools-25.1.6-py3.5.egg/pkg_resources/__init__.py", line 1512, in run_script
  File "/home/mcdonadt/miniconda3/envs/deblur/lib/python3.5/site-packages/deblur-0.0.1.dev0-py3.5.egg/EGG-INFO/scripts/deblur", line 19, in <module>
  File "/home/mcdonadt/miniconda3/envs/deblur/lib/python3.5/site-packages/skbio/__init__.py", line 13, in <module>
    import skbio.io  # noqa
  File "/home/mcdonadt/miniconda3/envs/deblur/lib/python3.5/site-packages/skbio/io/__init__.py", line 234, in <module>
    import_module('skbio.io.format.clustal')
  File "/home/mcdonadt/miniconda3/envs/deblur/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/mcdonadt/miniconda3/envs/deblur/lib/python3.5/site-packages/skbio/io/format/clustal.py", line 148, in <module>
    from skbio.alignment import TabularMSA
  File "/home/mcdonadt/miniconda3/envs/deblur/lib/python3.5/site-packages/skbio/alignment/__init__.py", line 205, in <module>
    from ._tabular_msa import TabularMSA
  File "/home/mcdonadt/miniconda3/envs/deblur/lib/python3.5/site-packages/skbio/alignment/_tabular_msa.py", line 14, in <module>
    import scipy.stats
  File "/home/mcdonadt/miniconda3/envs/deblur/lib/python3.5/site-packages/scipy/stats/__init__.py", line 344, in <module>
    from .stats import *
  File "/home/mcdonadt/miniconda3/envs/deblur/lib/python3.5/site-packages/scipy/stats/stats.py", line 173, in <module>
    import scipy.special as special
  File "/home/mcdonadt/miniconda3/envs/deblur/lib/python3.5/site-packages/scipy/special/__init__.py", line 636, in <module>
    from ._ufuncs import *
ImportError: libopenblasp-r0-39a31c03.2.18.so: cannot open shared object file: No such file or directory
```

**Comment by mortonjt** [comment]
That's actually a scikit-bio error.  Can you conda install scikit-bio on barnacle?

**Comment by wasade** [comment]
Yes, I have done so in other contexts.

**Pull request closed without merging**